src/test/scala/firrtlTests/FeatureSpec.scala

```diff
@@ -7,7 +7,7 @@ import org.scalatest._
 class FeatureSpec extends FirrtlPropSpec {
 
   property("Nested SubAccesses should be supported!") {
-    compileFirrtlTest("NestedSubAccessTester", "/features")
+    runFirrtlTest("NestedSubAccessTester", "/features")
```

Comment:
I agree the test should run rather than just compile, but this commit message seems to say the opposite?